accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -112,10 +123,310 @@ Windows-specific:
+    - `get_ErrorCode()`
+    - `get_Message()`
+    - `get_Source()`
+    - `ToString()`
+    - `get_Errors()`
+* **OleDbParameter**
+    - `.ctor()`
+    - `get_Direction()`
+    - `get_Value()`
+    - `set_Value(Object)`
+    - `System.ICloneable.Clone()`
+    - `ToString()`
+    - `.ctor(String, OleDbType)`
+    - `.ctor(String, OleDbType, Int32)`
+    - `.ctor(String, OleDbType, Int32, ParameterDirection, Boolean, Byte, Byte, String, DataRowVersion, Object)`
+    - `.ctor(String, OleDbType, Int32, ParameterDirection, Byte, Byte, String, DataRowVersion, Boolean, Object)`
+    - `.ctor(String, OleDbType, Int32, String)`
+    - `.ctor(String, Object)`
+    - `get_DbType()`
+    - `get_IsNullable()`
+    - `get_OleDbType()`
+    - `get_ParameterName()`
+    - `get_Precision()`
+    - `get_Scale()`
+    - `get_Size()`
+    - `get_SourceColumn()`
+    - `get_SourceColumnNullMapping()`
+    - `get_SourceVersion()`
+    - `ResetDbType()`
+    - `ResetOleDbType()`
+    - `set_DbType(DbType)`
+    - `set_Direction(ParameterDirection)`
+    - `set_IsNullable(Boolean)`
+    - `set_OleDbType(OleDbType)`
+    - `set_ParameterName(String)`
+    - `set_Precision(Byte)`
+    - `set_Scale(Byte)`
+    - `set_Size(Int32)`
+    - `set_SourceColumn(String)`
+    - `set_SourceColumnNullMapping(Boolean)`
+    - `set_SourceVersion(DataRowVersion)`
+* **OleDbParameterCollection**
+    - `Add(Object)`
+    - `Add(String, Object)`
+    - `Clear()`
+    - `Contains(Object)`
+    - `Contains(String)`
+    - `CopyTo(Array, Int32)`
+    - `get_Count()`
+    - `get_IsFixedSize()`
+    - `get_IsReadOnly()`
+    - `get_IsSynchronized()`
+    - `get_Item(Int32)`
+    - `get_Item(String)`
+    - `get_SyncRoot()`
+    - `GetEnumerator()`
+    - `IndexOf(Object)`
+    - `IndexOf(String)`
+    - `Insert(Int32, Object)`
+    - `Remove(Object)`
+    - `RemoveAt(Int32)`
+    - `Add(OleDbParameter)`
+    - `Add(String, OleDbType)`
+    - `Add(String, OleDbType, Int32)`
+    - `Add(String, OleDbType, Int32, String)`
+    - `AddRange(Array)`
+    - `AddRange(OleDbParameter[])`
+    - `AddWithValue(String, Object)`
+    - `Contains(OleDbParameter)`
+    - `CopyTo(OleDbParameter[], Int32)`
+    - `IndexOf(OleDbParameter)`
+    - `Insert(Int32, OleDbParameter)`
+    - `Remove(OleDbParameter)`
+    - `RemoveAt(String)`
+    - `set_Item(Int32, OleDbParameter)`
+    - `set_Item(String, OleDbParameter)`
+* **OleDbRowUpdatedEventArgs**
+    - `.ctor(DataRow, IDbCommand, StatementType, DataTableMapping)`
+    - `get_Command()`
+* **OleDbRowUpdatingEventArgs**
+    - `.ctor(DataRow, IDbCommand, StatementType, DataTableMapping)`
+    - `get_Command()`
+    - `set_Command(OleDbCommand)`
+* **OleDbSchemaGuid**
+    - `.ctor()`
+* **OleDbTransaction**
+    - `get_Connection()`
+    - `Begin()`
+    - `Begin(IsolationLevel)`
+    - `Commit()`
+    - `get_IsolationLevel()`
+    - `Rollback()`
+
 ### System.Diagnostics
 
 * **Process**
     - `EnterDebugMode()`
+    - `Kill(Boolean)`
```

Comment:
Is that because we can't walk the process tree?

Reply:
@adamsitnik @eiriktsarpalis

Reply:
Kill shouldn't be on this list.

Reply:
Removed